packages/graphql-language-service-server/src/GraphQLCache.ts

```diff
@@ -665,7 +665,9 @@ export class GraphQLCache implements GraphQLCacheInterface {
     const schemaKey = this._getSchemaCacheKeyForProject(
       projectConfig,
     ) as string;
-    schemaKey && this._schemaMap.delete(schemaKey);
```

Comment:
this is a valid expression as well? confused

Reply:
It’s valid but with `if` statement it’s more readable and clear